### PR TITLE
[wip] Remove `None` from usage rights publication list.

### PR DESCRIPTION
**Pull request by akash1810**
`None` is not a valid value - its was copied over when moving from a select to radio buttons.

### CODE REVIEW

File: kahuna/public/js/usage-rights/usage-rights-editor.html [hunk 1]
```diff
@@ -114,7 +97,7 @@
                         ng:model="ctrl.model[property.name]"
                         ng:required="property.required"
                         ng:options="o for o in ctrl.getOptionsFor(property)">
-                        <option value="">None</option>
+                        <option value="" disabled selected>None</option>
```
Comment (akash1810) [inline kahuna/public/js/usage-rights/usage-rights-editor.html]: setting as disabled makes the option behave more like a place holder value.

**Comment by jamesgorrie** [comment]
:+1:

File: kahuna/public/js/usage-rights/usage-rights-editor.html [hunk 2]
```diff
@@ -114,7 +97,7 @@
                         ng:model="ctrl.model[property.name]"
                         ng:required="property.required"
                         ng:options="o for o in ctrl.getOptionsFor(property)">
-                        <option value="">None</option>
+                        <option value="" disabled>None</option>
```
Comment (kenoir) [inline kahuna/public/js/usage-rights/usage-rights-editor.html]: Much nicer than the way I did it.

**Comment by akash1810** [comment]
@theefer @jamesgorrie can we talk about this today?

**Comment by akash1810** [comment]
Closing as too far behind master and is easier to create a new branch.

**Pull request closed without merging**